File: app/src/main/java/com/startogamu/zikobot/module/spotify_api/model/SpotifyArtist.java
package com.startogamu.zikobot.module.spotify_api.model;

import com.google.gson.annotations.SerializedName;

import lombok.Data;

/**
 * Created by josh on 01/04/16.
 */
@Data
public class SpotifyArtist {

    @SerializedName("href")
    private String href;
    @SerializedName("id")
    private String id;
    @SerializedName("name")
    private String name;
    @SerializedName("type")
    private String type;
    @SerializedName("uri")
    private String uri;
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/SpotifyArtist.kt
package com.joxad.zikobot.data.module.spotify_api.model

import com.google.gson.annotations.SerializedName

/**
 * Created by josh on 01/04/16.
 */
class SpotifyArtist {

    @SerializedName("href")
    val href: String? = null
    @SerializedName("id")
    val id: String? = null
    @SerializedName("name")
    val name: String? = null
    @SerializedName("type")
    val type: String? = null
    @SerializedName("uri")
    val uri: String? = null
    @SerializedName("images")
    val images: List<Image>? = null
}
